plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -256,6 +294,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns each block location on the 3D outer shell of the area.
         // This tag is useful for displaying particles or blocks to mark the boundary of the area.
+        // @Example
+        // # Plays the "TOTEM" effect in the shell of the area.
+        // - playeffect effect:TOTEM at:<cuboid[my_cuboid].shell>

```

Comment:
Again, testing, this time for the reason that the `playeffect` command has a bunch of weirdities and you've hit 2 or 3 of them in this example that you'd see if you tried it.